### DISCUSSION TITLE
Why this repository uses different gradle dependency implementation?

### DISCUSSION THREAD

**Discussion by Nephys222** [libre-tube/LibreTube#772] [Q&A] [answered]
I wonder why libretube uses such as:
`implementation libs.androidx.navigation.fragment
  implementation libs.androidx.navigation.ui...`
instead of 
`  implementation 'androidx.appcompat:appcompat:1.3.1'
    implementation 'androidx.cardview:cardview:1.0.0'
    implementation 'androidx.constraintlayout:constraintlayout:2.1.3'` ?

**Comment by Bnyro** [comment, marked as answer]
For better structure, libraries are located in gradle/libs.versions.toml

**Comment by Bnyro** [comment]
see https://docs.gradle.org/current/userguide/platforms.html#sub:conventional-dependencies-toml
